### PR TITLE
update dcbranch version

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

**Review by shifucun — APPROVED** [review]
thanks!

**Review by pradh — APPROVED** [review]

File: test/setup.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func setupInternal(
 		log.Fatalf("failed to create Bigquery client: %v", err)
 	}
 
-	branchTableName := "dcbranch_2022_03_06_18_18_49"
+	branchTableName := "dcbranch_2022_08_31_18_28_28"
```
Comment (pradh) [inline test/setup.go:109]: Wonder if we should use a "pinned" version?  Because when GC happens, this will fail in future...

Comment (shifucun) [reply test/setup.go:109]: I realized as we have newer non-branch cache, this branch cache is useless in the testing anymore. Let me try to send a PR to remove it and see if that works

Comment (Spaceenter) [reply test/setup.go:109]: OK - thanks @shifucun -- I'll deprecate this PR then.

**Pull request closed without merging**